config/config.go

```diff
@@ -412,6 +411,35 @@ func (c *ConsensusConfig) SetWalFile(walFile string) {
 	c.walFile = walFile
 }
 
+//-----------------------------------------------------------------------------
+// TxIndexConfig
+
+// TxIndexConfig defines the confuguration for the transaction
+// indexer, including tags to index.
+type TxIndexConfig struct {
+	// What indexer to use for transactions
+	//
+	// Options:
+	//   1) "null" (default)
+	//   2) "kv" - the simplest possible indexer, backed by key-value storage (defaults to levelDB; see DBBackend).
+	Indexer string `mapstructure:"indexer"`
+
+	// Comma-separated list of tags to index (by default the only tag is tx hash)
```

Comment:
If tx.hash is indexed by default, should we call them `IndexTags` `ExtraIndexTags` then?

Reply:
if i wanted to index txs the hit my account I might do `account=deadbeef` but if I want to index for all accounts, would I just do `account` ?

Reply:
oh boy :) we can do pattern-matching like "account.*". I just wanted to get smth simple done. If I am not mistaken, the func we have is enough for UI team.

Reply:
Is there a way to index everything by all tags ? Otherwise I don't think we have enough - ie. we need to be able to run nodes that index on behalf of all accounts, so an LCD can connect and query for all historical transactions related to its account

Reply:
> Is there a way to index everything by all tags ?

no, but we can add `*` - a wildcard

Reply:
I think we need too